modules/project-docs/pages/compatibility.adoc

```diff
@@ -80,6 +80,24 @@ M1 ARM  architecture is fully supported in the PHP SDK.
 AWS Amazon Graviton2, Apple M1 ARM processors, and ARMv8 on Ubuntu 20.04+ (from SDK 4.1).
 ****
 
+=== x86-32 Processor Support
+
+While the SDK can be compiled for 32-bit version of x86 platform, it is not
+generally tested and supported type of the deployment.
```

Comment:
We can remove the phrase "type of the deployment"
While the SDK can be compiled for 32-bit version of x86 platform, it is not generally tested and supported.